.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  extends: [
+    `@yarnpkg`,
+    `@yarnpkg/eslint-config/react`,
```

Comment:
```suggestion
```

Don't think this is needed as there is no react here

Reply:
It's used inside the website (e.g. https://github.com/arcanis/clipanion/blob/master/website/src/pages/index.js), as Docusaurus uses React.

Reply:
Ah, of course, never mind me :sweat_smile: